microprofile-jwt/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2020, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wildfly.quickstarts</groupId>
+        <artifactId>quickstart-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>20.0.0.Beta1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>microprofile-jwt</artifactId>
+    <packaging>war</packaging>
+    <name>JBoss PRODUCT_TYPE Quickstart: MicroProfile JWT</name>
```

Comment:
The standard module names are "Quickstart: artifactId", e.g. https://github.com/wildfly/quickstart/blob/master/contacts-jquerymobile/pom.xml#L33

Reply:
This format comes directly from the template https://github.com/wildfly/quickstart/blob/master/template/pom.xml#L25

Reply:
Yeah, looks like the template is outdated, i.e. 

:-)

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Quickstart: Parent 20.0.0.Beta1-SNAPSHOT:
[INFO]
[INFO] Quickstart: Parent ................................. SUCCESS [  0.090 s]
[INFO] Quickstart: app-client ............................. SUCCESS [  0.002 s]
[INFO] Quickstart: app-client - ejb ....................... SUCCESS [  0.018 s]
[INFO] Quickstart: app-client - client-simple ............. SUCCESS [  0.010 s]
[INFO] Quickstart: app-client - ear ....................... SUCCESS [  0.029 s]
[INFO] Quickstart: batch-processing ....................... SUCCESS [  0.036 s]
[INFO] Quickstart: bean-validation ........................ SUCCESS [  0.017 s]
...
[INFO] JBoss PRODUCT_TYPE Quickstart: MicroProfile JWT .... SUCCESS [  0.002 s]
[INFO] Quickstart: numberguess ............................ SUCCESS [  0.010 s]
[INFO] Quickstart: payment-cdi-event ...................... SUCCESS [  0.015 s]
[INFO] Quickstart: resteasy-jaxrs-client .................. SUCCESS [  0.011 s]
[INFO] Quickstart: security-domain-to-domain .............. SUCCESS [  0.003 s]
[INFO] Quickstart: security-domain-to-domain - ejb ........ SUCCESS [  0.010 s]
[INFO] Quickstart: security-domain-to-domain - web ........ SUCCESS [  0.015 s]
[INFO] Quickstart: security-domain-to-domain - ear ........ SUCCESS [  0.005 s]
[INFO] Quickstart: servlet-async .......................... SUCCESS [  0.015 s]
[INFO] Quickstart: servlet-filterlistener ................. SUCCESS [  0.013 s]
[INFO] Quickstart: servlet-security ....................... SUCCESS [  0.013 s]
[INFO] Quickstart: shopping-cart .......................... SUCCESS [  0.001 s]
[INFO] Quickstart: shopping-cart - server ................. SUCCESS [  0.008 s]
[INFO] Quickstart: shopping-cart - client ................. SUCCESS [  0.008 s]
[INFO] Quickstart: spring-greeter ......................... SUCCESS [  0.019 s]
[INFO] Quickstart: spring-kitchensink-basic ............... SUCCESS [  0.025 s]
[INFO] Quickstart: spring-kitchensink-springmvctest ....... SUCCESS [  0.022 s]
[INFO] Quickstart: spring-resteasy ........................ SUCCESS [  0.012 s]
[INFO] Quickstart: tasks-jsf .............................. SUCCESS [  0.016 s]
[INFO] Quickstart: tasks-rs ............................... SUCCESS [  0.015 s]
[INFO] Quickstart: temperature-converter .................. SUCCESS [  0.012 s]
[INFO] Quickstart: thread-racing .......................... SUCCESS [  0.028 s]
[INFO] Quickstart: websocket-client ....................... SUCCESS [  0.013 s]
[INFO] Quickstart: websocket-endpoint ..................... SUCCESS [  0.014 s]
[INFO] Quickstart: websocket-hello ........................ SUCCESS [  0.009 s]
[INFO] Quickstart: wicket-ear ............................. SUCCESS [  0.001 s]
[INFO] Quickstart: wicket-ear - ejb ....................... SUCCESS [  0.009 s]
[INFO] Quickstart: wicket-ear - war ....................... SUCCESS [  0.019 s]
[INFO] Quickstart: wicket-ear - ear ....................... SUCCESS [  0.004 s]
[INFO] Quickstart: wicket-war ............................. SUCCESS [  0.019 s]
[INFO] Quickstart: xml-jaxp ............................... SUCCESS [  0.018 s]
[INFO] Quickstart: jts .................................... SUCCESS [  0.002 s]
[INFO] Quickstart: jts - application-component-2 .......... SUCCESS [  0.012 s]
[INFO] Quickstart: jts - application-component-1 .......... SUCCESS [  0.014 s]
[INFO] Quickstart: ejb-remote ............................. SUCCESS [  0.001 s]
[INFO] Quickstart: ejb-remote - server-side ............... SUCCESS [  0.007 s]
[INFO] Quickstart: ejb-remote - client .................... SUCCESS [  0.010 s]
[INFO] Quickstart: jta-crash-rec .......................... SUCCESS [  0.011 s]
[INFO] Quickstart: wsat-simple ............................ SUCCESS [  0.016 s]
[INFO] Quickstart: wsba-coordinator-completion-simple ..... SUCCESS [  0.012 s]
[INFO] Quickstart: wsba-participant-completion-simple ..... SUCCESS [  0.012 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.461 s
[INFO] Finished at: 2020-03-17T14:37:54+01:00
[INFO] ------------------------------------------------------------------------
```

Reply:
Opened https://github.com/wildfly/quickstart/pull/392 with the tempate fix.